vlib/orm/README.md

```diff
@@ -12,7 +12,7 @@
 - `[unique]` sets the field as unique
 - `[unique: 'foo']` adds the field to a unique group
 - `[skip]` field will be skipped
-- `[sql: type]` sets the type which is used in sql (special type `serial`)
+- `[sql: type]` sets the v type or (special type `serial`) which is used in sql
```

Comment:
Hm, I do not think that is clearer 🤔 ... May be it needs usage examples instead. @JalonSolov what do you think?

Reply:
Hmm...  Perhaps something like
```
`[sql: type]` where `type` is a V type such as `int` or `f64`, or special type `serial`
```